ang/civicase/ContactCard.js

```diff
@@ -31,6 +31,16 @@
         $document.bind('click', closeDropdownIfClickedOutside);
       }());
 
+      /**
+       * Returns the cached information for the given contact.
+       *
+       * @param {String} contactID
+       * @return {Object}
+       */
+      scope.getContact = function (contactID) {
```

Comment:
Are we testing this function?

Reply:
I removed the function because it was not necessary (after removing the `ng-init="..."`). I added a test for the new code, but it does not check the DOM directly because the information is displayed inside the uib popover and I was not able to display it in the test. But this should suffice for now.